pkg/registry/file/storage_test.go

```diff
@@ -275,12 +275,7 @@ func TestStorageImpl_Get(t *testing.T) {
 			},
 			content: "{}",
 			create:  true,
-			want: &v1beta1.SBOMSPDXv2p3{
-				TypeMeta: v1.TypeMeta{
-					APIVersion: "spdx.softwarecomposition.kubescape.io/v1beta1",
-					Kind:       "SBOMSPDXv2p3",
-				},
-			},
```

Comment:
question: do we really not want the type meta when creating an empty file?

Reply:
no, it's not needed, this was a remain when I force-added them to solve the patches (but ultimately didn't work)

Reply:
Checked more, and with your explanation it seems to make sense: you get an empty object, you write an empty object, you get an empty object back. It should not be the job of the storage to make sense of the object, unless really needed.